tests/e2e/machine_pool_test.go

```diff
@@ -71,5 +69,50 @@ var _ = Describe("TF Test", func() {
 				Expect(respBody.ID()).To(Equal(mp_out.Name))
 			})
 		})
+		Context("Author:amalykhi-High-OCP-64905 @OCP-64905 @amalykhi", func() {
```

Comment:
You are not labeling the case with day2, and importance. Better to also label it with feature, so in future we can trigger the CI's regression by feature.

Reply:
@xueli181114 Do you suggest the  feature label like _ci.machinepool_ or more detailed?